docs/src/lib/methods_fix.md

```diff
@@ -0,0 +1,41 @@
+# Functions with several methods
```

Comment:
what is the purpose of the file `methods_fix.md`?

Reply:
If you add it, all the `Documenter` warnings about missing documentation is gone. Of course this is dangerous, so I commented this line. But it can help find missing docs at least for new function/type names quickly. In particular, I finally managed to exclude the warnings about `apply_recipe`!

Reply:
hooray 🙌 

most if not all of these functions with several methods should be documented, such as `constraints_list`. why does Documenter complain? since we include some (maybe not all) the variations of `constraints_list` in the docs.

Reply:
No, the reason is described in #72. `Documenter` just does not recognize methods with default values or `where` syntax. Currently there is just no other way than manually searching for those docs at the moment. That is why I said that it is dangerous :worried: